brave-spancollector-scribe/src/test/java/com/github/kristofa/brave/scribe/ScribeSpanCollectorTest.java

```diff
@@ -93,7 +82,39 @@ public void testCollectWithDefaultAnnotation() throws TTransportException {
         final BinaryAnnotation binaryAnnotation = binary_annotations.get(0);
         assertEquals(KEY1, binaryAnnotation.getKey());
         assertEquals(VALUE1, new String(binaryAnnotation.getValue()));
+    }
 
+    @Test
+    public void testNotifyOfSpanCollectionEvents() throws TTransportException {
```

Comment:
good start on a feature test. This is where we can explain what the callbacks mean